### ISSUE TITLE
Problem when install @stencil/core 0.0.6-14

### ISSUE THREAD

**Issue by wellbeck190** [ionic-team/stencil#239]
```
npm ERR! Linux 4.4.0-93-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v6.11.4
npm ERR! npm  v3.10.10
npm ERR! path /home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename

npm ERR! enoent ENOENT: no such file or directory, rename '/home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648' -> '/home/r0/Workspace/my-app/node_modules/@stencil/core/node_modules/@types/node'
npm ERR! enoent ENOENT: no such file or directory, rename '/home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648' -> '/home/r0/Workspace/my-app/node_modules/@stencil/core/node_modules/@types/node'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/r0/Workspace/my-app/npm-debug.log
```
This problem occurred when I install node module for stencil-app-starter. I think it is problem of stencil core.

**Comment by jgw96** [comment]
Hey @wellbeck190 , in your package.json if you pin your @stencil/core version to 0.0.6-10 and do npm install does it work?

**Comment by wellbeck190** [comment]
Stencil-app-starter dependency:
```
  "dependencies": {
    "@stencil/core": "0.0.6-10",
    "@stencil/router": "latest"
  },
```
Npm install error:
```
      ├── require-main-filename@1.0.1 
      ├── set-blocking@2.0.0 
      ├─┬ string-width@2.1.1 
      │ └── is-fullwidth-code-point@2.0.0 
      ├── which-module@2.0.0 
      ├── y18n@3.2.1 
      └── yargs-parser@7.0.0 

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.1 (node_modules/sane/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! Linux 4.4.0-93-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v6.11.4
npm ERR! npm  v3.10.10
npm ERR! path /home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename

npm ERR! enoent ENOENT: no such file or directory, rename '/home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648' -> '/home/r0/Workspace/my-app/node_modules/@stencil/core/node_modules/@types/node'
npm ERR! enoent ENOENT: no such file or directory, rename '/home/r0/Workspace/my-app/node_modules/.staging/@types/node-daaf9648' -> '/home/r0/Workspace/my-app/node_modules/@stencil/core/node_modules/@types/node'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/r0/Workspace/my-app/npm-debug.log
```
I am using Ubuntu 16.04.

I continue comeback previous version until work:
0.0.6-7 haven't this problem but problem with npm run build.
... same problem
0.0.6-4 work well and I can build ionic/core.

@jgw96

**Comment by thusitha92** [comment]
I am having the same error mentioned above in Windows 10 x64.

```
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! path G:\stencil\my-app\node_modules\.staging\@types\node-1679b34c
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall rename

npm ERR! enoent ENOENT: no such file or directory, rename 'G:\stencil\my-app\node_modules\.staging\@types\node-1679b34c' -> 'G:\stencil\my-app\node_modules\@stencil\core\node_modules\@types\node'
npm ERR! enoent ENOENT: no such file or directory, rename 'G:\stencil\my-app\node_modules\.staging\@types\node-1679b34c' -> 'G:\stencil\my-app\node_modules\@stencil\core\node_modules\@types\node'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
```

**Comment by LloydVincent** [comment]
Problem occurred for me also. (Windows 10, node@v6.9.0)
- **Bugged**: npm@3.10.8
- **Works**: npm@5.5.1

**Comment by thusitha92** [comment]
@LloydVincent I was able to fix this error by updating node version from v6.9.1 to v8.7.0 [Latest]

**Comment by mlabieniec** [comment]
+1 

Same error for me on mac os 10.13 node version v6.11.3, npm v3.10.10

Works on node v8.5.0, npm v5.3.0

**Comment by jgw96** [comment]
Hello all! I was able to reproduce this with npm 3 and can confirm that with npm 5 it works fine.

**Comment by azawawi** [comment]
The following workaround fixed the problem on my Ubuntu 16.04, nodejs `v6.11.4`.

```
sudo npm install npm@latest -g
```

**Comment by jgw96** [comment]
Hello all! With Node 8 and therefore NPM 5 becoming the LTS release today I am going to close this issue for now. Also, we are removing the bundling of dependencies that we were doing which is what was causing this issue with npm < 5. Thanks for using Stencil!
